packages/teleport/src/Player/ProgressBar/Slider/Slider.jsx

```diff
@@ -625,7 +650,9 @@ THE SOFTWARE.
     },
 
     _getValueFromPosition: function (position) {
-      var diffValue = position / (this.state.sliderLength - this.state.handleSize) * (this.props.max - this.props.min);
+      var diffValue =
+        (position / (this.state.sliderLength - this.state.handleSize)) *
+        (this.props.max - this.props.min);
```

Comment:
i checked this was the same (also i learned that multiplication and division is whichever comes first not necessarily PEMDAS)